### PR TITLE
Blog fix -  Encrypting Data With SSH Keys and Golang

### PR DESCRIPTION
**Pull request by msali123**
http://localhost:4002/blog/encrypting-data-with-ssh-keys-and-golang/
-	Needs file names added where makes sense - Done
-	Remove `alex@earthly:~/$` prompts and use `$` instead - Done
-	Separate command line output from input – tried to separate one

### CODE REVIEW

**Comment by msali123** [comment]
Line 65 : 1z8l6QfFo9nTBlYI0k= alex@earthly

Please check if I have to remove this from the output code block as well?

**Comment by msali123** [comment]
@jalletto

File: blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md [hunk 1]
```diff
@@ -40,19 +46,29 @@ Let's start with what I knew, generating a new RSA key with ssh-keygen
...
+~~~
 
 Perfect, we can then display my public key with:
 
-```
-    alex@earthly:~/$ cat /tmp/testkey.pub 
-    ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDBRl0U4mwO/jQ7kYJidSnQy0ci45j1QZ1do7NEC/08cG0jbNCSX6mblFr0JWruLpp6Z1WA/BL+GngCwATBeEt7dSAHNpOvT0fJ4roWv6/KmOLOCjKq26a0MvMf1g/YFa5tP5Zi7UW5Hp4vGCTXRPyywNJvh1/cHKuq2j79fUX+4cG9p01a1Y89/a3Q7L5UkB4JoFuaA9sVzVg4H5A2vRVR/pEIRRFuPuxHDVcNblA6CsKFf0zBoLatXv+aBn86dX8EtwB13HdRsKq+XmBwnWJiS+Cz1GBhnKf4LM/Ca46qy2ExQnOOt49COUOoU6DI7P5bf4I33pNDDLoTvFFKzyXWTRgwg1tiyiRzfIjO+mg0kQM/dZ7+M8W49AQv+MR8Uh0bykECXn6u8yEibEgInYlj0ziWXtf6lPEg+505hDTLlvPWXpo8nLluR5COwgFVSbNcMnY9o3KHeog598mQxiqrXWWbGmra7SgXrKmqJGqUbkZqH1z8l6QfFo9nTBlYI0k= alex@earthly
-```
+~~~{.bash caption=">_"}
+    $:~/$ cat /tmp/testkey.pub
```
Comment (adamgordonbell) [inline blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md:54]: Prompt should but `$`
```suggestion
    $ cat /tmp/testkey.pub 
```

File: blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md [hunk 2]
```diff
@@ -40,19 +46,29 @@ Let's start with what I knew, generating a new RSA key with ssh-keygen
...
+    po8nLluR5COwgFVSbNcMnY9o3KHeog598mQxiqrXWWbGmra7SgXrKmqJGqUbkZqH \
+    1z8l6QfFo9nTBlYI0k= alex@earthly
+~~~
 
 and since we're all friends here, I'll share my example private key (you should never share your key with anyone, I'm only sharing this as an example -- I won't ever be using this key anywhere).
 
-```
-    alex@earthly:~/$ cat /tmp/testkey
+~~~{.bash caption=">_"}
+    $:~/$ cat /tmp/testkey
```
Comment (adamgordonbell) [inline blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md:71]:
```suggestion
    $ cat /tmp/testkey
```

**Review by adamgordonbell — CHANGES_REQUESTED** [review]
I couple small changes. Otherwise great!

File: blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md (hunk 2 above)
Comment (msali123) [reply blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md]: Done

File: blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2020-12-11-encrypting-data-with-ssh-keys-and-golang.md]: Done

**Review by adamgordonbell — APPROVED** [review]

**Comment by msali123** [comment]
@jalletto

**Pull request merged**